### PR TITLE
Edit xgboost_customer_churn_studio.ipynb

### PR DESCRIPTION
**Pull request by annbech**
*Description of changes: Edit pass for readability

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws_sagemaker_studio/getting_started/xgboost_customer_churn_studio.ipynb [hunk 1]
```diff
@@ -109,13 +109,13 @@
     "---\n",
     "## Data\n",
     "\n",
-    "Mobile operators have historical records on which customers ultimately ended up churning and which continued using the service. We can use this historical information to construct train an ML model that can predict customer churn. After training the model, we can pass the profile information of an arbitrary customer (the same profile information that we used to train the model) to the model, and have the model predict whether this customer is going to churn.\n",
+    "Mobile operators' records show which customers ended up churning and which continued using the service. We can use this historical information to train an ML model that can predict customer churn. After training the model, we can pass the profile information of an arbitrary customer (the same profile information that we used to train the model) to the model, and have the model predict whether this customer will churn.\n",
```
Comment (ericangelokim) [inline aws_sagemaker_studio/getting_started/xgboost_customer_churn_studio.ipynb]: Minor:

Mobile operator records

File: aws_sagemaker_studio/getting_started/xgboost_customer_churn_studio.ipynb [hunk 2]
```diff
@@ -109,13 +109,13 @@
...
+    "The dataset that we use is publicly available and was mentioned in the book [Discovering Knowledge in Data](https://www.amazon.com/dp/0470908742/) by Daniel T. Larose. It's attributed by the author to the University of California Irvine Repository of Machine Learning Datasets. The downloaded and preprocessed dataset is in the `data` folder that accompanies this notebook. It's been split into training and validation datasets. To see how the dataset was preprocessed, see this [XGBoost customer churn notebook that starts with the original dataset](https://github.com/awslabs/amazon-sagemaker-examples/blob/master/introduction_to_applying_machine_learning/xgboost_customer_churn/xgboost_customer_churn.ipynb).  \n",
     "\n",
-    "We will train on a csv file without the header, but for now, the following cell uses `pandas` to load a bit of the data from a version of the training data that has a header. \n",
+    "We'll train on a CSV file without the header. But for now, the following cell uses `pandas` to load some of the data from a version of the training data that has a header. \n",
```
Comment (ericangelokim) [inline aws_sagemaker_studio/getting_started/xgboost_customer_churn_studio.ipynb]: Minor:

We'll train on a CSV file without the header, but for now the following cell uses ```pandas``` to load some ...

File: aws_sagemaker_studio/getting_started/xgboost_customer_churn_studio.ipynb [hunk 3]
```diff
@@ -431,9 +431,9 @@
...
+    "#### Trial 2 - XGBoost in framework mode\n",
     "\n",
-    "For the next trial, we'll train a similar model, but using XGBoost in framework mode. This way of using XGBoost should be to familiar to users who have worked with the open source XGBoost. Using XGBoost as a framework provides more flexibility than using it as a built-in algorithm as it enables more advanced scenarios that allow pre-processing and post-processing scripts to be incorporated into your training script. Specifically, we will be able to specify a list of rules that we want the SageMaker Debugger to evaluate our training process against."
+    "For the next trial, we'll train a similar model, but we'll use XGBoost in framework mode. If you've worked with the open source XGBoost framework, this way of using XGBoost should be  familiar to you. Using XGBoost as a framework provides more flexibility than using it as a built-in algorithm because it enables more advanced scenarios that allow you to incorporate preprocessing and post-processing scripts into your training script. Specifically, we'll be able to specify a list of rules that we want Debugger to evaluate our training process against."
```
Comment (ericangelokim) [inline aws_sagemaker_studio/getting_started/xgboost_customer_churn_studio.ipynb]: Minor:

...we'll train a similar model, but use XGBoost in framework mode.

Do you mean script mode? We should use common terminology.

**Comment by iyerr3** [comment]
@annbech Please fix the conflicts to merge the PR.

**Review by aaronmarkham — APPROVED** [review]

**Pull request merged**